include/ocpp/v201/charge_point.hpp

```diff
@@ -520,8 +523,6 @@ class ChargePoint : public ChargePointInterface, private ocpp::ChargingStationBa
     /// device model
     void remove_network_connection_profiles_below_actual_security_profile();
 
-    void handle_message(const EnhancedMessage<v201::MessageType>& message);
-
```

Comment:
Should this be in the PR description? Or did you mean somewhere else?

Reply:
Ideally, this would be in:
- the related commit messages *and*
- summarized in the PR description

An example from my NREL project: https://github.com/e-mission/e-mission-phone/pull/1138